docs/lib/datastore/fragments/flutter/getting-started/50_initDataStore.md

```diff
@@ -1,12 +1,10 @@
 To initialize the Amplify DataStore, use the `Amplify.addPlugin()` method to add the AWS DataStore Plugin. We also need to import the codegen dart file `ModelProvider.dart`
 
 ```dart
-import 'codegen/ModelProvider.dart';
+import 'models/ModelProvider.dart';
 
 // Add this in your app initialization
-AmplifyDataStore datastorePlugin =
-    AmplifyDataStore(modelProvider: ModelProvider.instance);
-Amplify.addPlugin(datastorePlugin);
+Amplify.addPlugin(AmplifyDataStore(modelProvider: ModelProvider.instance));
```

Comment:
Actually think that the `datastorePlugin` declaration is more self-expressive even if it is slightly more verbose.